docs/index.rst

```diff
@@ -488,6 +488,28 @@ Fields can also be of type ``Callable``:
     callable, no validation of arguments, their types or the return
     type is performed.
 
+Color Type
+..........
+
+You can use ``Color`` data type for storing colors as per
+`CSS3 specification <http://www.w3.org/TR/css3-color/#svg-color>`_.
+Color can be defined as:
+
+- `name <http://www.w3.org/TR/SVG11/types.html#ColorKeywords>`_ (e.g. ``Black``, ``azure``)
+- `hexadecimal value <https://en.wikipedia.org/wiki/Web_colors#Hex_triplet>`_ (e.g. ``0x000``, ``#FFFFFF``, ``7fffd4``)
+- RGB/RGBA tuple (e.g. ``(255, 255, 255)``, ``(255, 255, 255, 1.0)``
+- RGB/RGBA string (e.g. ``"rgb(255, 255, 255)"``)
+
+.. warning::
+
+    RGBA tuple or string with alpha channel not equal to ``1.0`` will never be
+    a valid CSS3 color as no `alpha bleding <https://en.wikipedia.org/wiki/Alpha_compositing#Alpha_blending>`_
```

Comment:
typo "alpha bleding" should probably be "alpha blending"

Reply:
Thanks!